core/core-impl/src/main/java/ai/timefold/solver/core/impl/domain/solution/descriptor/SolutionDescriptor.java

```diff
@@ -838,6 +839,12 @@ public void validateConstraintWeight(String constraintPackage, String constraint
     // Extraction methods
     // ************************************************************************
 
+    public Collection<Object> getAllEntities(Solution_ solution) {
```

Comment:
Is this method used somewhere in the new code?

Reply:
No, will delete.